pkg/notifications/github_issues.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2024 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This file details the named fields of OID extensions supported by Fulcio.
+// A list of OID extensions supported by Fulcio can be found here:
+// https://github.com/sigstore/fulcio/blob/main/docs/oid-info.md
+// Named fields in this file have been imported from this file in the Fulcio repository:
+// https://github.com/sigstore/fulcio/blob/main/pkg/certificate/extensions.go
+// Updates to the Fulcio repository extensions file should be matched here accordingly and vice-versa.
+
+package notifications
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/google/go-github/v65/github"
+	"github.com/sigstore/rekor-monitor/pkg/identity"
+)
+
+var (
+	notificationPlatformGitHubIssueBodyHeaderText = "Rekor-monitor found the following pairs of monitored identities and matching log entries: "
+	notificationPlatformGitHubIssueLabels         = []string{"rekor-monitor", "automatically generated"}
```

Comment:
What happens if the labels don't exist in the repo? Do we need to document that we assume these labels exist, or are the labels created automatically?

Reply:
from testing, it seems that the labels are created automatically